{"name": "test_login_phone_error_case", "status": "passed", "attachments": [{"name": "log", "source": "9d0d1ef3-6eb0-4f95-ac7f-fd98644c6913-attachment.txt", "type": "text/plain"}], "start": 1597503501940, "stop": 1597503503982, "uuid": "f0c1f47c-e124-417e-bb60-8451f65a5f5b", "historyId": "210797769f28844d15197db9e5f676ed", "testCaseId": "7bcb06678aeec0849c5a6efc7c1a2546", "fullName": "case.login.test_login_case.TestLoginCase#test_login_phone_error_case", "labels": [{"name": "tag", "value": "login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "case.login"}, {"name": "suite", "value": "test_login_case"}, {"name": "subSuite", "value": "TestLoginCase"}, {"name": "host", "value": "FYYX-2019VXOSDQ"}, {"name": "thread", "value": "20080-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.login.test_login_case"}]}